src/types.ts

```diff
@@ -111,13 +111,58 @@ export interface WapeSaleConfig {
   };
 }
 
+export interface GenSaleConfig {
+  /**
+   * Address of the sale contract
+   */
+  contractAddress: string;
+
+  /**
+   * URI for the merkle tree file
+   */
+  merkleTreeFileUri: string;
+
+  /**
+   * web3 provider to access blockchain with (on read operations)
+   */
+  web3Provider: ethers.providers.Provider;
+
+  /**
+   * amount the SDK should return for the maximum amount purchasable in a single transaction in private sale
+   */
+  privateSaleTransactionLimit?: number;
```

Comment:
I can't find where this variable is using, please remove it if not used.

Reply:
removed